lib/topological_inventory/persister/workflow.rb

```diff
@@ -127,6 +128,8 @@ def send_changes_to_queue!
         }
 
         message[:payload] = persister.inventory_collections.select { |x| x.name }.index_by(&:name).transform_values! do |x|
+          next if x.name == :service_instance_tasks
```

Comment:
Wondering if this would be necessary since this collection probably wont have created/updated/deleted records (haven't confirmed)
If we end up having to add any more dependent collections this list will end up growing

Reply:
this collection have updated_records filled by custom_save (core's PR). usually it contains only ID, but tasks contains more info for the 2nd kafka topic, so this is not about dependent collections as a whole, but this specific case.

Reply:
Ah okay too bad, maybe a comment then explaining why we're skipping this one?

Reply:
okay it'll be useful